package.json

```diff
@@ -30,5 +30,9 @@
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
     "prepublish": "npm run build"
+  },
+  "dependencies": {
```

Comment:
I agree but right now the demo is inside of the app, so `npm start` depends on having access to `react`... Maybe extracting the demo directory might be a good solution, with this we can add it as `peerDependency` to `react-slot-fill` and as `dependency` to the demo.